### PR TITLE
tiup: add instruction to custom exist offline mirror

### PR DESCRIPTION
**Pull request by AstroProfundis**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Add instructions of how to customize exist offline mirror with TiUP. Typically useful when the user is using an offline mirror and want to upgrade TiUP itself without upgrading TiDB components.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by lilyjazz — APPROVED** [review]
LGTM

**Comment by TomShawn** [comment]
@AstroProfundis Assignees in docs-cn PRs mean the translator who is assigned to translate the PR to docs. Do you want to be the translator?😀

**Comment by TomShawn** [comment]
@lucklove PTAL

**Comment by AstroProfundis** [comment]
> @AstroProfundis Assignees in docs-cn PRs mean the translator who is assigned to translate the PR to English. Do you want to be the translator?grinning

Ummm, no ><

**Review by lucklove — APPROVED** [review]
LGTM

File: production-deployment-using-tiup.md [hunk 1]
```diff
@@ -133,6 +133,56 @@ aliases: ['/docs-cn/dev/production-offline-deployment-using-tiup/', '/zh/tidb/de
 
         此时，`tidb-community-server-${version}-linux-amd64.tar.gz` 就是一个独立的离线环境包。
 
+3. 自定义制作的离线镜像，或调整已有离线镜像中的内容
+
+    如果从官网下载的离线镜像不满足具体需求，或者希望对已有的离线镜像内容进行调整，例如增加某个组件的新版本等，可以使用如下方式进行：
```
Comment (TomShawn) [inline production-deployment-using-tiup.md:138]:
```suggestion
    如果从官网下载的离线镜像不满足你的具体需求，或者希望对已有的离线镜像内容进行调整，例如增加某个组件的新版本等，可以采取以下步骤进行操作：
```

File: production-deployment-using-tiup.md [hunk 2]
```diff
@@ -133,6 +133,56 @@ aliases: ['/docs-cn/dev/production-offline-deployment-using-tiup/', '/zh/tidb/de
...
+
+    1. 在制作离线镜像时，可通过参数指定具体的组件和版本等信息，获得不完整的离线镜像。例如，制作一个只包括 v1.5.2 版本 TiUP 和 TiUP Cluster 的离线镜像可通过如下命令完成：
```
Comment (TomShawn) [inline production-deployment-using-tiup.md:140]:
```suggestion
    1. 在制作离线镜像时，可通过参数指定具体的组件和版本等信息，获得不完整的离线镜像。例如，要制作一个只包括 v1.5.2 版本 TiUP 和 TiUP Cluster 的离线镜像，可执行如下命令：
```

File: production-deployment-using-tiup.md [hunk 3]
```diff
@@ -133,6 +133,56 @@ aliases: ['/docs-cn/dev/production-offline-deployment-using-tiup/', '/zh/tidb/de
...
+
+        {{< copyable "shell-regular" >}}
+
+        ```bash
+        tiup mirror clone tiup-custom-mirror-v1.5.2 --tiup v1.5.2 --cluster v1.5.2
+        ```
+
+        如果只需要某一特定平台的组件，也可以通过 `--os` 和 `--arch` 参数来指定。
+
+    2. 参考上文中的方式，将此不完整的离线镜像传输到隔离环境的中控机
```
Comment (TomShawn) [inline production-deployment-using-tiup.md:150]:
```suggestion
    2. 参考上文“使用 TiUP 制作离线镜像”第 2 步的方式，将此不完整的离线镜像传输到隔离环境的中控机。
```

File: production-deployment-using-tiup.md [hunk 4]
```diff
@@ -133,6 +133,56 @@ aliases: ['/docs-cn/dev/production-offline-deployment-using-tiup/', '/zh/tidb/de
...
+
+    3. 在隔离环境的中控机上，查看当前使用的离线镜像路径
+
+        较新版本的 TiUP 可以直接通过命令获取当前的镜像地址：
```
Comment (TomShawn) [inline production-deployment-using-tiup.md:154]:
```suggestion
    3. 在隔离环境的中控机上，查看当前使用的离线镜像路径。较新版本的 TiUP 可以直接通过命令获取当前的镜像地址：
```

File: production-deployment-using-tiup.md [hunk 5]
```diff
@@ -133,6 +133,56 @@ aliases: ['/docs-cn/dev/production-offline-deployment-using-tiup/', '/zh/tidb/de
...
+
+        {{< copyable "shell-regular" >}}
+
+        ```bash
+        tiup mirror show
+        ```
+
+        如果提示 `show` 命令不存在，可能当前使用的是较老版本的 TiUP, 此时可以通过查看 `$HOME/.tiup/tiup.toml` 获得正在使用的镜像地址。
+
+        将此镜像地址记录下来，后续步骤中将以变量 `${base_mirror}` 指代此镜像地址。
```
Comment (TomShawn) [inline production-deployment-using-tiup.md:164]:
```suggestion
        以上命令如果提示 `show` 命令不存在，可能当前使用的是较老版本的 TiUP。此时可以通过查看 `$HOME/.tiup/tiup.toml` 获得正在使用的镜像地址。将此镜像地址记录下来，后续步骤中将以变量 `${base_mirror}` 指代此镜像地址。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**